src/instances.cr

```diff
@@ -75,14 +75,32 @@ spawn do
         client.connect_timeout = 5.seconds
         client.read_timeout = 5.seconds
         begin
-          stats = JSON.parse(client.get("/api/v1/stats").body)
+          req = client.get("/api/v1/stats")
+          stats = JSON.parse(req.body)
+
+          api = false
+          cors = false
+          req = client.get("/api/v1/trending")
+          if req.status_code == 200
+            begin
+              if req.headers["Access-Control-Allow-Origin"] == "*"
+                cors = true
+              end
+
+              trending = JSON.parse(req.body)
+              trending[0]["videoId"].as_s
```

Comment:
I guess that this part is meant to validate the data returned and trigger an exception if the data can't be parsed? If yes, can you add a comment explaining that?